### ISSUE TITLE
YAML value string misinterpreted as float

### ISSUE THREAD

**Issue by mikemartin** [statamic/v2-hub#791]
### Steps to reproduce

1) Add value "2e02325309" to text field using the CP. (Note the "e" in the number string)
2) Click Save

---

After saving, you should see this error:

```
[2016-06-29 19:47:21] dev.ERROR: UnexpectedValueException: The Response content must be a string or object implementing __toString(), "boolean" given. in /Users/mike/Sites/trychameleon/statamic/vendor/symfony/http-foundation/Response.php:399
Stack trace:
```

**Comment by edalzell** [comment]
hey @mikemartin, does this still happen? I'm trying to reproduce.

**Comment by mikemartin** [comment]
@edalzell Also appears to be fixed since 2.1.19.
